test/setup-beam.test.js

```diff
@@ -875,11 +875,31 @@ describe('.getVersionFromSpec(_)', () => {
     got = setupBeam.getVersionFromSpec(spec, matrix.otp['ubuntu-20.04'])
     assert.deepStrictEqual(got, expected)
 
+    spec = 'maint-24'
+    expected = 'maint-24'
+    got = setupBeam.getVersionFromSpec(spec, matrix.otp['ubuntu-22.04'])
+    assert.deepStrictEqual(got, expected)
+
+    spec = '> 0'
+    expected = 'OTP-26.2.5'
```

Comment:
Below we can see that `latest` is with RC, but considering a range is without it.